### DISCUSSION TITLE
Yt command no present after the go update

### DISCUSSION THREAD

**Discussion by isudoajl** [danielmiessler/Fabric#1014] [General]
Hello,

After the go fabric update, I just figure out that yt command it's not present anymore. There is something wrong?

**Comment by mseigal** [comment]
It has been replaced with fabric -y as an argument in the fabric command so there is no need to use a separate go application.

**Comment by isudoajl** [comment]
Thanks for your time. I had already updated fabric with go, but the -y option did not appear, but after doing update again it does appear now.

go install -ldflags "-X main.version=$(git describe --tags --always)" github.com/danielmiessler/fabric@latest
